src/components/02_atoms/MultipleFields/MultipleFields.js

```diff
@@ -171,13 +171,13 @@ class MultipleFields extends Component {
       state: { newItemAdded },
       props: { label, value: values, component, onChange },
     } = this;
-    const usedValues = values.length ? values : [this.createEmptyItem()];
+    const usedValues = values || [this.createEmptyItem()];
```

Comment:
Are you sure about this particular change? An empty array is trueish: ```> const a = [];
undefined
> !!a;
true```

Reply:
Yes, I have tested that. We are either receiving the `values` as `null` or an array with length > 0
Do you want me to add the array validation?